misc/assets/inforequest-flow.svg

```diff
@@ -0,0 +1,343 @@
+<svg version="1.1" viewBox="0 0 1070 1165" xmlns="http://www.w3.org/2000/svg" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  <style>
+      text, foreignObject {
+        font-family: sans-serif;
+      }
+      .title {
+        font-weight: bold;
+        font-size: 16pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .label {
+        font-size: 10pt;
+        dominant-baseline: hanging;
+      }
+      .action {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: 100%;
+        text-align: center;
+      }
+      .caption {
+        font-weight: bold;
+        font-size: 12pt;
+      }
+      .note {
+        font-size: 10pt;
+      }
+      .legend {
+        font-size: 10pt;
+        padding: 5pt;
+      }
+      .arrow {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+        marker-end: url(#arrow);
+      }
+  </style>
+  <defs>
+    <marker id="start" orient="auto" markerWidth="4" markerHeight="4" refX="2" refY="2">
+      <circle cx="2" cy="2" r="2" />
+    </marker>
+    <marker id="arrow" orient="auto" markerWidth="8" markerHeight="6" refX="8" refY="3">
+      <path d='M 0,0 V 6 L 8,3 Z' />
+    </marker>
+    <g id="action">
+      <path d="M -100,-15 -85,-30 85,-30 100,-15 100,15 85,30 -85,30 -100,15 Z" fill="none" stroke="black" stroke-width="1pt" />
+    </g>
+    <g id="end">
+      <ellipse cx="0" cy="0" rx="20" ry="20" fill="none" stroke="black" stroke-width="1pt" />
+      <text text-anchor="middle" dominant-baseline="middle">⊥</text>
+    </g>
+  </defs>
+
+  <path d="M 250,0 v 1125" fill="none" stroke="black" stroke-width="1pt" stroke-dasharray="2pt" />
+  <path d="M 650,0 v 1125" fill="none" stroke="black" stroke-width="1pt" stroke-dasharray="2pt" />
+
+  <g transform="translate(100 20)">
```

Comment:
V predchadzajucich modeloch sme pouzivali horne a spodne oramovanie na 100 bodov.

Reply:
@viliambalaz Good point. Opravil som to v dalsom commite.